frontend/src/modules/staff/components/orders/OrderDialog.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Button,
+} from "@mui/material";
+
+interface OrderDialogProps {
+  open: boolean;
+  handleCloseDialog: () => void;
+  confirmRemoveOrReject: () => void;
+}
+
+const OrderDialog: React.FC<OrderDialogProps> = ({
+  open,
+  handleCloseDialog,
+  confirmRemoveOrReject,
+}) => {
+  return (
+    <Dialog
+      open={open}
```

Comment:
Can we use the reusable [baseDialog](https://github.com/UPRMeals/UPRMeals/blob/main/frontend/src/shared/components/baseDialog.tsx) here? There's a couple of examples in the code, puedes buscar ejemplos en el `modules/staff/components` folder: https://github.com/UPRMeals/UPRMeals/tree/main/frontend/src/modules/staff/components.